packages/runtime/container-runtime/src/containerRuntime.ts

```diff
@@ -2206,7 +2217,7 @@ export class ContainerRuntime extends TypedEventEmitter<IContainerRuntimeEvents>
             wrapSummaryInChannelsTree(summarizeResult);
             pathPartsForChildren = [channelsTreeName];
         }
-        this.addContainerStateToSummary(summarizeResult);
+        this.addContainerStateToSummary(summarizeResult, fullTree, trackState);
```

Comment:
We might want to pass in the telemetryContext here

Reply:
Or is this a purposeful oversight?

Reply:
Good catch.